src/DIRAC/ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -27,21 +27,21 @@ def getUsernameForDN(dn, usersList=None):
         if not result["OK"]:
             return result
         usersList = result["Value"]
-    for username in usersList:
-        if dn in gConfig.getValue(f"{gBaseRegistrySection}/Users/{username}/DN", []):
-            return S_OK(username)
-    return S_ERROR("No username found for dn %s" % dn)
+    for userName in usersList:
+        if dn in gConfig.getValue(f"{gBaseRegistrySection}/Users/{userName}/DN", []):
+            return S_OK(userName)
+    return S_ERROR("No user name found for dn %s" % dn)
 
 
-def getDNForUsername(username):
+def getDNForUsername(userName):
```

Comment:
I still think this isn't needed `username` is a word so there is no need to camel case it as if it was `user name`.

It's also unrelated to this PR and is arguably a breaking API change.

Reply:
I am not an expert in English of course. Username is a single word ? What about usergroup, userjob, usertype, etc ? In the DIRAC code we use consistently userName almost everywhere. Many other places including the calls to Registry functions.

Reply:
> Username is a single word ?

* https://en.wiktionary.org/wiki/username
* https://www.merriam-webster.com/dictionary/username
* https://www.oxfordlearnersdictionaries.com/definition/english/username
* https://www.collinsdictionary.com/dictionary/english/username

> Many other places including the calls to Registry functions.

There are more cases of `username`, including in function names (I can't find any functions with `[uU]serName` in the name).

Reply:
Hmm... Only a slight win of userName over username in the DIRAC code. 

     $ grep -r serName * | wc
     418    2292   45056
     $ grep -r sername * | grep -v 'sername"' | wc
     394    2276   43389

So, cases like:

     def getUserGroupIds(self, userName, userGroup, insertIfMissing=True):

should become:
 
     def getUserGroupIds(self, username, userGroup, insertIfMissing=True):

Which one is more consistent ?

Reply:
Interesting.

Anyway I think it means that there's no benefit in changing, in this PR, from `username` to `userName`.

Reply:
It is already changed. Is there a benefit to change it back ? We have half code using username and another half using userName. I have just learnt that username can be regarded as a single word according to the modern English norms. I think that both cases are equivalent and the choice is a matter of taste. "username" is probably better as we use it as keyword in many places and also external libraries/protocolss, e.g. jwts', use "username" as keywords. Then, probably, it should be converted consistently everywhere. This particular case is nowhere special.

Reply:
I rolled this back - is it OK now to finalize the review ?